# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1503469061 25200
#      Tue Aug 22 23:17:41 2017 -0700
# Node ID 712736d43124183e64ef96c581b5e43dba0f2672
# Parent  e3a36f84d01d1c54cc6c7a336920395e2dee672f
do not include octave.jar in the source distribution

* scripts/java/module.mk (scripts_EXTRA_DIST): Remove
$(%canon_reldir%_JAR_FILES) from the list.

diff --git a/scripts/java/module.mk b/scripts/java/module.mk
--- a/scripts/java/module.mk
+++ b/scripts/java/module.mk
@@ -54,14 +54,13 @@ endif
 
 FCN_FILES += $(%canon_reldir%_FCN_FILES)
 
 PKG_ADD_FILES += %reldir%/PKG_ADD
 
 DIRSTAMP_FILES += %reldir%/$(octave_dirstamp)
 
 scripts_EXTRA_DIST += \
-  $(%canon_reldir%_JAR_FILES) \
   $(%canon_reldir%_JAVA_SRC)
 
 scripts_CLEANFILES += \
   $(%canon_reldir%_JAR_FILES) \
   $(%canon_reldir%_JAVA_CLASSES)
